dluhc-component-library/src/components/siteSelectionForm/components/DynamicForm.tsx

```diff
@@ -11,14 +13,17 @@ interface DynamicFormProps {
 }
 
 enum InputType {
-  Input,
+  TextInput,
+  NumberInput,
   MultiSelect,
   None,
 }
 
 const getInputType = (formPageSchema: FormPageSchema) => {
   if (formPageSchema.type === "string") {
-    return formPageSchema.enum ? InputType.MultiSelect : InputType.Input;
+    return formPageSchema.enum ? InputType.MultiSelect : InputType.TextInput;
+  } else if (formPageSchema.type === "number") {
```

Comment:
We dont need the "else" here, just do...

```
if (formPageSchema.type === "number") {
  return InputType.NumberInput;
}
```
below the first if.

The returns mean we dont fall through to the next one and its easier to read.